toolkit/data-sources/db-sync/src/db_model.rs

```diff
@@ -367,6 +375,53 @@ pub(crate) async fn get_token_utxo_for_epoch(
 		.await?)
 }
 
+#[cfg(feature = "governed-map")]
+pub(crate) async fn get_changes(
+	pool: &Pool<Postgres>,
+	address: &Address,
+	last_block: Option<BlockNumber>,
+	new_block: BlockNumber,
```

Comment:
I propose "after_block" (or "since_block", but "after" is more precise to me, because "since" isn't that clear about inclusion) and "to_block" instead of "last_block" and "new_block" in this layer.